src/sentry/models/rule.py

```diff
@@ -57,6 +58,8 @@ class Rule(Model):
         choices=RuleSource.as_choices(),
     )
     owner = FlexibleForeignKey("sentry.Actor", null=True, on_delete=models.SET_NULL)
+    user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="SET_NULL")
+    team = FlexibleForeignKey("sentry.Team", null=True, on_delete=models.SET_NULL)
```

Comment:
I think these should be `owner_user_id` and `owner_team_id`.

We should also add a constraint to guarantee that only one is set.

Reply:
Sounds good, I've made those changes now.